lib/Maintenance/Tasks/AssetDocumentConversionTask.php

```diff
@@ -50,9 +51,15 @@ public function execute()
 
             try {
                 if ($asset instanceof Asset\Document) {
-                    $this->logger->debug(sprintf('Processing document with ID %s | Path: %s', $asset->getId(),
-                        $asset->getFullPath()));
+                    $this->logger->debug(sprintf('Processing document with ID %s | Path: %s', $asset->getId(), $asset->getFullPath()));
                     $asset->processPageCount();
+                    $asset->save();
+
+                    $searchEntry = Data::getForElement($asset);
+                    if ($searchEntry instanceof Data and $searchEntry->getId() instanceof Data\Id) {
+                        $searchEntry->setDataFromElement($asset);
+                        $searchEntry->save();
+                    }
```

Comment:
`$asset->save();` also generate the search entry, right?

Reply:
You're totally right, `save()` does already the job :)